redbot/core/core_commands.py

```diff
@@ -1672,12 +1672,13 @@ async def contact(self, ctx: commands.Context, *, message: str):
     @commands.command()
     @checks.is_owner()
     async def dm(self, ctx: commands.Context, user_id: int, *, message: str):
-        """Sends a DM to a user
+        """Sends a DM to a user.
 
-        This command needs a user id to work.
-        To get a user id enable 'developer mode' in Discord's
-        settings, 'appearance' tab. Then right click a user
-        and copy their id"""
+        This command needs a User ID to work.
```

Comment:
User ID, should be user ID. 

For both instances you used it . Other than that it's looking fine to me but flame is the grammar freak